squeal-postgresql/src/Squeal/PostgreSQL/Type/Schema.hs

```diff
@@ -210,7 +210,7 @@ data TableConstraint
   = Check [Symbol]
   | Unique [Symbol]
   | PrimaryKey [Symbol]
-  | ForeignKey [Symbol] Symbol [Symbol]
+  | ForeignKey [Symbol] (Symbol, Symbol) [Symbol]
```

Comment:
might be cleaner without the pair?

Reply:
I think it makes sense to keep the pair and create some sugar like `"sch0" ! "tab"`

Reply:
I already have `:::` and `:=>` for type level pairs, trying not to add _too_ many new type operators

Reply:
Right, well I'll leave the choice up to you, but I think its more intuitive to make it as similar to the `#sch ! #tab` syntax as possible.